mogc_partnerships/views.py

```diff
@@ -177,7 +180,7 @@ def get_serializer(self, *args, **kwargs):
 
         return super(CohortMembershipCreateView, self).get_serializer(*args, **kwargs)
 
-    def get_serializer_context(self):
+    def get_cohort(self):
```

Comment:
Note: this is temporary and will use [this lib function](https://github.com/academic-innovation/mogc-partnerships/pull/33/files#diff-1c0e548606ec30bd21d05024245a1188b8a059f67f854e8629c5b34858f792fdR6) when that PR is merged.

Reply:
Should be merged now, if you want to replace it!

Reply:
🙌